daemon/stop.go

```diff
@@ -23,6 +23,9 @@ func (daemon *Daemon) ContainerStop(name string, seconds *int) error {
 	}
 	if !container.IsRunning() {
 		err := fmt.Errorf("Container %s is already stopped", name)
```

Comment:
I remember 304 typically returns empty body (for HTTP caching purpose). Not sure the string of `Container %s is already stopped` or `Container %s is not running` is part of the HTTP response.

Maybe we should return an empty message (as 304 already indicates `StatusNotModified`)?